src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,57 +70,176 @@ contract DssSpellCollateralAction {
+
+        // Set the stability fee
+        DssExecLib.setIlkStabilityFee(ilk, RWA007_A_RATE, /* _doDrip = */ false);
+
+        // Set collateralization ratio
+        DssExecLib.setIlkLiquidationRatio(ilk, RWA007_A_MAT);
+
+        // Poke the spotter to pull in a price
+        DssExecLib.updateCollateralPrice(ilk);
+
+        // Give the urn permissions on the join adapter
+        DssExecLib.authorize(MCD_JOIN_RWA007_A, RWA007_A_URN);
+
+        // MCD_PAUSE_PROXY and Monetalis permission on URN
+        RwaUrnLike(RWA007_A_URN).hope(address(this));
+        RwaUrnLike(RWA007_A_URN).hope(address(RWA007_A_OPERATOR));
+
+        // MCD_PAUSE_PROXY and Monetails permission on RWA007_A_OUTPUT_CONDUIT
+        RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).hope(address(this));
+        RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).mate(address(this));
+        RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).hope(RWA007_A_OPERATOR);
+        RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).mate(RWA007_A_OPERATOR);
+        // Coinbase custody whitelist for URN destination address
+        RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).kiss(address(RWA007_A_COINBASE_CUSTODY));
+
+        // MCD_PAUSE_PROXY and Monetails permission on RWA007_A_INPUT_CONDUIT_URN
+        RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_URN).mate(address(this));
+        RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_URN).mate(RWA007_A_OPERATOR);
+        // Set "quitTo" address for RWA007_A_INPUT_CONDUIT_URN
+        RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_URN).file("quitTo", RWA007_A_INPUT_CONDUIT_QUIT_TO);
+
+        // MCD_PAUSE_PROXY and Monetails permission on RWA007_A_INPUT_CONDUIT_JAR
+        RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_JAR).mate(address(this));
+        RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_JAR).mate(RWA007_A_OPERATOR);
+        // Set "quitTo" address for RWA007_A_INPUT_CONDUIT_JAR
+        RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_JAR).file("quitTo", RWA007_A_INPUT_CONDUIT_QUIT_TO);
+
+        // Add RWA007 contract to the changelog
+        CHANGELOG.setAddress("RWA007",                     RWA007);
+        CHANGELOG.setAddress("PIP_RWA007",                 pip);
+        CHANGELOG.setAddress("MCD_JOIN_RWA007_A",          MCD_JOIN_RWA007_A);
+        CHANGELOG.setAddress("RWA007_A_URN",               RWA007_A_URN);
+        CHANGELOG.setAddress("RWA007_A_JAR",               RWA007_A_JAR);
+        CHANGELOG.setAddress("RWA007_A_OUTPUT_CONDUIT",    RWA007_A_OUTPUT_CONDUIT);
+        CHANGELOG.setAddress("RWA007_A_INPUT_CONDUIT_URN", RWA007_A_INPUT_CONDUIT_URN);
+        CHANGELOG.setAddress("RWA007_A_INPUT_CONDUIT_JAR", RWA007_A_INPUT_CONDUIT_JAR);
+
+        // Add RWA007 to ILK REGISTRY
+        REGISTRY.put(
+            ilk,
+            MCD_JOIN_RWA007_A,
+            RWA007,
+            decimals,
+            RWA007_REG_CLASS_RWA,
+            pip,
+            address(0),
+            "RWA007-A: Monetalis",
```

Comment:
Ilk Registry Name doesn't look matching the one specified in the technical assessment (see [here](https://forum.makerdao.com/t/rwa007-mip65-monetalis-clydesdale-ces-domain-team-assessment/17787))

Reply:
done